File: amuta/amuta-crm/Seldat.Amuta.Web/src/app/students/shared/models/student-payment.ts
import { Student } from './student.model';

export class StudentPayment {
    constructor(
       public Student:Student=null ,
       public Year:number=0 ,
       public Month:number=0 ,
       public PaymentSum:number=0 ,
       public AttendanceSum:number=0,
       public ExamsSum:number=0 ,
       public StateBudgetSum:number=0 ,
       public HealthSupportSum:number=0 ,
       public DentalHealthSupportSum:number=0 ,
       public FinancialSupportSum:number=0 ,
       public LoansGivenSum:number=0,
       public loansReturnSum:number=0
    ){}
   
}

File: amuta/amuta-crm/Seldat.Amuta.Entities/StudentPayment.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Seldat.Amuta.Entities
{
    public class StudentPayment
    {
        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        [Required]
        public Student Student { get; set; }
        public int Year { get; set; }
        public int Month { get; set; }
        public int PaymentSum { get; set; }
        public int AttendanceSum { get; set; }
        public int ExamsSum { get; set; }
        public int StateBudgetSum { get; set; }
        public int HealthSupportSum { get; set; }
        public int DentalHealthSupportSum { get; set; }
        public int FinancialSupportSum { get; set; }
        public int LoansGivenSum { get; set; }
        public int LoansReturnSum { get; set; }

    }
}
